src/script/event/EventService.ts

```diff
@@ -127,18 +127,30 @@ export class EventService {
 
     try {
       if (this.storageService.db) {
-        const entry = await this.storageService.db
+        let entry = await this.storageService.db
           .table(StorageSchemata.OBJECT_STORE.EVENTS)
           .where('id')
           .equals(eventId)
           .filter(record => record.conversation === conversationId)
           .first();
+        if (!entry) {
+          entry = await this.storageService.db
+            .table(StorageSchemata.OBJECT_STORE.EVENTS)
+            .where('conversation')
+            .equals(conversationId)
+            .filter(item => item.data.replacing_message_id === eventId)
+            .first();
+        }
         return entry;
       }
 
       const records = (await this.storageService.getAll(StorageSchemata.OBJECT_STORE.EVENTS)) as EventRecord[];
       return records
-        .filter(record => record.id === eventId && record.conversation === conversationId)
+        .filter(
+          record =>
+            (record.id === eventId && record.conversation === conversationId) ||
+            (record.id === eventId && record.data.replacing_message_id === eventId),
+        )
```

Comment:
not sure how important this is.

Reply:
It's for when we don't have a dexie DB under the hood (not sure if we have valid scenarios where we have a different engine... 🤔 ). But let's keep it there 👍

Reply:
For reference, it was introduced here with temporary guest devices https://github.com/wireapp/wire-webapp/pull/7086/files#diff-6ab223d7041cd0190d856a1ee68a04f4c60175878ff0fcc77810a789569eeff7